skimage/util/_coord_conv.py

```diff
@@ -0,0 +1,234 @@
+import numpy as np
+
+
+def xy_to_rc(points=None, image=None):
+    """Convert points' coordinates and/or image from `x, y` to `r, c` convention.
+
+        o--->....       o--->....
+        |   x   .       |   c   .
+        |       .  ==>  |       .
+        v y     .       v r     .
+        .........       .........
+
+    Parameters
+    ----------
+    points : (K, 2) ndarray, optional
+        Array of points' `x, y` coordinates.
+    image : (M, N[, C]) ndarray, optional
+        Image stored in `x, y` axes convention.
+
+    Returns
+    -------
+    points_out : (K, 2) ndarray
+        Array of `points` `r, c` coordinates.
+    image_out : (M, N[, C]) ndarray
+        Same data as `image`, but in `r, c` convention.
+    """
+    if points is not None:
+        points_out = np.fliplr(points)
```

Comment:
since both inputs are optional and a user may accidentally ask for `xy_to_rc(image):`
I would ass some asserts `assert np.asarray(points).ndim == 2 and np.asarray(points).shape[1] == 2`